Mammoth/TSE/CAccessibilitySettings.cpp

```diff
@@ -0,0 +1,76 @@
+#include "PreComp.h"
+#include "TSEAccessibilitySettings.h"
+#include <sstream>
+
+CAccessibilitySettings::CAccessibilitySettings(void) {
+	m_rgbColorPlayer = CG32bitPixel(255, 255, 255);
+	m_rgbColorEscort = CG32bitPixel(80, 200, 200);
+	m_rgbColorFriendly = CG32bitPixel(80, 255, 80);
+	m_rgbColorNeutral = CG32bitPixel(80, 80, 255);
+	m_rgbColorEnemy = CG32bitPixel(255, 80, 80);
+	m_rgbColorAngry = CG32bitPixel(230, 150, 80);
+	m_rgbColorProjectile = CG32bitPixel(255, 255, 0);
+};
+CAccessibilitySettings::~CAccessibilitySettings(void) {};
+
+void CAccessibilitySettings::SetIFFColor (CString sSettingsValue, int iIFFType) {
+	std::stringstream hex_ss;
```

Comment:
You might be better off calling LoadRGBColor here (which is defined in TSE/Utilities.cpp)

If LoadRGBColor doesn't support the specific syntax you're looking for then, enhance it.

Reply:
Added to LoadRGBColor, but also switched to image editor style hex codes too as the defaults in the settings file, so the functionality is there if anyone enters RGB codes using programmer style hex ('0x') just not mandatory to use